changelogs/custom-fields-api-v1

```diff
@@ -0,0 +1,38 @@
+---
+title: Custom Fields API
+type: added
+---
+
+Released stable version v1 of Custom Fields API.
+
+The Custom Fields API provides a way to define new fields in the Trackunit data model. Allowing to extend and customize Trackunit Manager. In this version 1 we support extending the data model of assets, accounts, groups and sites with new fields.
+
+Below all available endpoints:
+
+To define a custom field, you first need to [add a definition using the API.](/reference/custom-field-definitions)
+
+> ➕ GET: /custom-fields/v1/definitions
+[Get definitions](ref:getdefinitionsforaccount-1)
```

Comment:
It seems the links somehow changed. Now it is: 
```suggestion
[Get definitions](ref:custom-fields-get-definitions)
```

I haven't checked the others. We did some updates to the docs so probably why it changed.